src/wacksbywarby/wack.py

```diff
@@ -51,7 +53,7 @@ def announce_new_sales(discord: Discord, sales: list[Sale], num_total_sales: int
             color = None
         # format and send the discord message
         sold_out = sale.quantity == 0
-        quantity_message = f" ({sale.num_sold} of 'em)" if sale.num_sold > 1 else ""
+        quantity_message = f" ({sale.num_sold:.0f} of 'em)" if sale.num_sold > 1 else ""
```

Comment:
what does thsi change do?

Reply:
`num_sold` became a number instead of a string. so it used to say "2 of 'em", but when it became a number, it started saying "2.0 of 'em". this converts the number to not have any decimal places so it can go back to saying "2 of 'em"